Demo/src/main/java/com/paypal/android/api/services/SDKSampleServerAPI.kt

```diff
@@ -59,12 +57,6 @@ class SDKSampleServerAPI {
         @GET("/client_id")
         suspend fun fetchClientId(): ClientId
 
-        @PATCH("/orders/{orderId}")
```

Comment:
Oh I need to remove this too. I forgot that patch was just for NativeCheckout.

Reply:
Yeah it couldn't hurt. There was an actual compilation issue on the Android side though. We were referencing a type that was owned by the Native XO sdk.